cli/up.go

```diff
@@ -0,0 +1,26 @@
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	upCmd = &cobra.Command{
+		Use:   "up <group> [scale]",
+		Short: "Bring a group up",
+		Long:  "Submit a group, with an optional scale, and start it",
+		Run:   upRun,
+	}
+)
+
+func upRun(cmd *cobra.Command, args []string) {
+	submitRun(cmd, args)
```

Comment:
error during submit:
```
core@core-01 ~ $ ls -R
.:
example  inagoctl

./example:
example-unit.service
core@core-01 ~ $ cat ./example/example-unit.service
[Unit]
Description=Inago Demo Unit

[Service]
ExecStart=/bin/bash -c "while true; do echo Hi; sleep 10; done"
core@core-01 ~ $ ./inagoctl up example 4
2016-03-31 10:22:23.337 | ERROR    | context.Background: [{/usr/code/.gobuild/src/github.com/giantswarm/inago/cli/submit.go:47: } {/usr/code/.gobuild/src/github.com/giantswarm/inago/cli/submit.go:80: invalid scale: must be 1 for unscalable groups}]
core@core-01 ~ $ ./inagoctl status example
2016-03-31 10:22:29.651 | ERROR    | context.Background: Failed to find group 'example'.
```
so, if submit errors and exits - which it seems to do on all errors, we're fine

for already existing slices:
```
core@core-01 ~ $ ls -R
.:
example  inagoctl

./example:
example-unit@.service
core@core-01 ~ $ cat ./example/example-unit\@.service
[Unit]
Description=Inago Demo Unit

[Service]
ExecStart=/bin/bash -c "while true; do echo Hi; sleep 10; done"
core@core-01 ~ $ ./inagoctl submit example
2016-03-31 10:20:55.241 | INFO     | context.Background: Succeeded to submit group 'example'.
core@core-01 ~ $ ./inagoctl status example
Group        Units  FDState  FCState  SAState   IP            Machine

example@b5a  *      loaded   loaded   inactive  172.17.8.103  2eb8b3bb62894c269df58a00248a68eb

core@core-01 ~ $ ./inagoctl up example
2016-03-31 10:21:09.618 | INFO     | context.Background: Succeeded to submit group 'example'.
2016-03-31 10:21:14.625 | INFO     | context.Background: Succeeded to start 2 slices for group 'example': [899 b5a].
core@core-01 ~ $ ./inagoctl status example
Group        Units  FDState   FCState   SAState  IP            Machine

example@899  *      launched  launched  active   172.17.8.102  49e7e432625f4d9fa3e5071cf3f9c26e
example@b5a  *      launched  launched  active   172.17.8.103  2eb8b3bb62894c269df58a00248a68eb
```
uncertain about the behaviour here - it is logical (it's the same as two submits, and then a start), and i'm not certain how we'd get around it anyway.